docs/user/hbase/install.rst

```diff
@@ -23,50 +25,110 @@ and untar it somewhere convenient:
     $ ls /opt/geomesa
     bin/  conf/  dist/  docs/  examples/  lib/  LICENSE.txt  logs/
 
-Configuring HBase and Hadoop Dependencies
------------------------------------------
+Configuration and Classpaths
+----------------------------
+
+GeoMesa HBase requires Hadoop and HBase jars and configuration files to be available on the classpath. This includes
+files such as the hbase-site.xml and core-site.xml files in addition to standard jars and libraries. Configuring the
+classpath is important if you plan to use the GeoMesa HBase command line tools to ingest and manage GeoMesa.
+
+By default, GeoMesa HBase will attempt to read various HBase and Hadoop related environmental variables in order to
+build the classpath. You can configure environment variables and classpath settings in
+``geomesa-hbase_2.11-$VERSION/conf/geomesa-env.sh`` or in your external env (e.g. bashrc file). The logic GeoMesa
+uses to determine which external entries to include on the classpath is:
+
+    1. If the environmental variables ``GEOMESA_HADOOP_CLASSPATH`` and ``GEOMESA_HBASE_CLASSPATH`` are set then GeoMesa
+    HBase will use these variables to set the classpath and skip all other logic.
+
+    2. Next, if ``$HBASE_HOME`` and ``$HADOOP_HOME`` are set then GeoMesa HBase will attempt to build the classpath by
+    searching for jar files and configuration in standard locations...Note that this is very specific to the
+    installation or distribution of Hadoop you are using and may not be reliable.
+
+    3. If no environmental variables are set but the ``hbase`` and ``hadoop`` commands are available then GeoMesa will
+    interrogate them for their classpaths by running the ``hadoop classpath`` and ``hbase classpath`` commands.
```

Comment:
maybe add a sentence that this will be slightly slow, which is why it's 3rd priority